frontend/src/pages/distributedWorkloads/GlobalDistributedWorkloadsRoutes.tsx

```diff
@@ -20,7 +20,7 @@ const GlobalDistributedWorkloadsRoutes: React.FC = () => {
         .map((tab) => (
           <Route
             key={tab.id}
-            path={`${tab.path}${tab.projectSelectorMode !== null ? '/:namespace?' : ''}`}
+            path={`${tab.path}/:namespace`}
```

Comment:
We do still need the `?` on `${tab.path}/:namespace?`, sorry, my mistake. It tells react-router that `:namespace` is optional here when matching the route (so that it will still render `GlobalDistributedWorkloads` when there's no namespace in the URL, which will catch that and redirect us). I had it confused with the `?` delimiter on the end of a URL for query params.